core/src/test/java/org/apache/iceberg/io/TestResolvingIO.java

```diff
@@ -19,8 +19,16 @@
 package org.apache.iceberg.io;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatCode;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
```

Comment:
I believe these imports are wrong and should be `...Assertions.assertThatThrownBy` rather than `AssertionsForClassTypes`

Reply:
Thanks, I fixed that